checkout-service/public/index.html

```diff
@@ -6,7 +6,7 @@
   </head>
   <body>
     <div id='checkout'></div>
-    <script type='text/javascript' src='checkout_bundle.js'></script>
+    <script type='text/javascript' src='https://fec-blitva-checkout.s3.us-east-2.amazonaws.com/checkout_bundle.js.gz'></script>
```

Comment:
Just curious about your decision to use the compressed bundle file in the script tag as opposed to using [express to serve file](https://stackoverflow.com/questions/55748479/how-to-use-highly-compressed-bundle-file-bundle-tar-gz-for-hosting-in-server)? I didn't find much out there regarding best practice, but most I found suggested to use express (possibly for cross-browser compatibility?). I'm not certain and was curious about your thoughts as I am using gz compressed files for optimization as well!

Reply:
Hey Justin! So I think server side rendering with express might be best. For this project however, I wanted for the bundle to be available at all times without having to keep my instance up. Serving the bundle with S3 allows me to do so.

Reply:
Makes perfect sense, thank you for sharing!